skyrimnet_chatterbox/chatterbox/models/s3gen/matcha/flow_matching.py

```
@@ -1,129 +1,129 @@
+        for step in range(1, len(t_span)):
+            dphi_dt = self.estimator(x, mask, mu, t, spks, cond)
+
+            x = x + dt * dphi_dt
+            t = t + dt
+            sol.append(x)
+            if step < len(t_span) - 1:
+                dt = t_span[step + 1] - t
+
+        return sol[-1]
+
+    def compute_loss(self, x1, mask, mu, spks=None, cond=None):
```

Ruff (0.14.10)

Unused method argument: `cond`

(ARG002)